frontend/public/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -50,6 +50,21 @@ export const OperatorHubSubscribeForm = withFormState((props: OperatorHubSubscri
   }
 
   const {provider, channels = [], packageName} = props.packageManifest.data.status;
+  const srcProvider = _.get(props.packageManifest, 'metadata.labels.opsrc-provider');
+  let providerType;
+    switch (srcProvider) {
```

Comment:
All these catalog views seem to require a lot of "normalizing" and hard-coded display name converting...

Reply:
I think the priority is to get these features in and we can always enhance later. But I agree we need to come up with a smoother flow